command/sidechain/registration/register_validator.go

```diff
@@ -0,0 +1,236 @@
+package registration
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/command/helper"
+	"github.com/0xPolygon/polygon-edge/command/polybftsecrets"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi"
+	bls "github.com/0xPolygon/polygon-edge/consensus/polybft/signer"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/wallet"
+	"github.com/0xPolygon/polygon-edge/contracts"
+	"github.com/0xPolygon/polygon-edge/secrets"
+	"github.com/0xPolygon/polygon-edge/txrelayer"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/spf13/cobra"
+	"github.com/umbracle/ethgo"
+)
+
+var (
+	stakeManager         = contracts.ValidatorSetContract
+	stakeFn              = contractsapi.ChildValidatorSet.Abi.Methods["stake"]
```

Comment:
`stakeFn` and `registerFn` can be part of `contractsabi` as I mentioned in the previous comment.

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1258/commits/323027a56e7b07a3eb9c9c25816e8226bbcfee5a